File: MOBILE/TrimfitMobile/app/src/main/java/pl/smartica/trimfitmobile/api/model/Employee.kt
package pl.smartica.trimfitmobile.api.model

import org.json.JSONObject

class Employee {
    var employeeId: Int? = null
    var employeeFirstName : String? = null
    var employeeLastName : String? = null

    constructor(item: JSONObject){
        employeeId = item.getInt("employee_Id")
        employeeFirstName = item.getString("employee_Firstname")
        employeeLastName = item.getString("employee_Lastname")
    }
}


/*    "employee": {
      "employee_Id": 0,
      "employee_Firstname": "string",
      "employee_Lastname": "string",
      "employee_Birthdate": "2019-09-08T10:35:44.336Z",
      "employee_Gender": 0,
      "employee_Added": "2019-09-08T10:35:44.336Z",
      "position_Id": [
        0
      ],*/

File: API/Trimfit.Domain/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Trimfit.Domain.Abstractions;

namespace Trimfit.Domain.Models
{
    [Table("employee")]
    public class Employee : IDatabaseEntity
    {
        [Key]
        [Column("employee_id")]
        public int Id { get; set; }

        [Column("employee_firstname")]
        public string Firstname { get; set; }

        [Column("employee_lastname")]
        public string Lastname { get; set; }

        [Column("employee_birthdate")]
        public DateTime Birthdate { get; set; }

        [Column("employee_gender")]
        public int Gender { get; set; }

        [Column("employee_added")]
        public DateTime Added { get; set; }

        [Column("position_id")]
        public List<int> PositionId { get; set; }

        [Column("employee_status")]
        public int Status { get; set; }

        [Column("address_id")]
        [ForeignKey("Address")]
        public int? AddressId { get; set; }
        public virtual Address Address { get; set; }

        [Column("employee_display_name")]
        public string DisplayName { get; set; }
        [Column("employee_photo_url")]
        public string PhotoUrl { get; set; }

        [Column("user_id")]
        public int? UserId { get; set; }
        public User User { get; set; }
        public ICollection<Club> Clubs { get; set; }
    }
}
